### PR TITLE
Avoid NPE in case where an app handler is unset

### PR DESCRIPTION
**Pull request by brenthdaniel**

### CODE REVIEW

**Comment by brenthdaniel** [comment]
#build

File: dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java [hunk 1]
```diff
@@ -118,8 +118,14 @@ public StartAction(ApplicationConfig config,
     @Override
     public void execute(ExecutorService executor) {
         _startTime.set(System.currentTimeMillis());
+        @SuppressWarnings({ "rawtypes" })
+        ApplicationHandler handler = _aii.getHandler();
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java:122]: I would make this `final`

File: dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java [hunk 2]
```diff
@@ -128,13 +134,11 @@ public void execute(ExecutorService executor) {
...
-                AppMessageHelper.get(_aii.getHandler()).audit("APPLICATION_SLOW_STARTUP", _config.getName(), TimestampUtils.getElapsedTime(_startTime.get()));
+                AppMessageHelper.get(handler).audit("APPLICATION_SLOW_STARTUP", _config.getName(), TimestampUtils.getElapsedTime(_startTime.get()));
             }
         }, maxWait, TimeUnit.SECONDS));
 
         try {
-            @SuppressWarnings("rawtypes")
-            ApplicationHandler handler = _aii.getHandler();
             @SuppressWarnings("unchecked")
             ApplicationMonitoringInformation ami = handler.setUpApplicationMonitoring(_aii);
```
Comment (tjwatson) [inline dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/statemachine/StartAction.java:143]: I don't see how an NPE is prevented here when `handler==null`. Did you intend to return early from the `if (handler == null)` check above?

**Review by tjwatson — CHANGES_REQUESTED** [review]

**Comment by brenthdaniel** [comment]
#build

**Review by tjwatson — APPROVED** [review]

**Comment by brenthdaniel** [comment]
#run-libby-bot

**Pull request merged**